styles/snackbar.css

```diff
@@ -0,0 +1,102 @@
+.snackbar-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 3rem;
+    width: 900px;
+    min-width: 100vh;
+    margin-left: 3rem;
+}
+
+.button-container {
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+    margin: 2rem;
+}
+
+.button-container>button {
+    border: none;
+    background-color: white;
+    color: rgb(0, 0, 0);
+    padding: 0.5rem;
+    border-radius: 5px;
+    transition: 0.5s;
+}
+.button-container>button:hover{
+    background-color: rgb(100, 100, 100);
+    color: rgb(255, 255, 255);
```

Comment:
Any specific reason for using colors directly rather than using them with variables?

Reply:
use  color variables